airbyte-integrations/connectors/source-redshift/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/RedshiftSourceAcceptanceTest.java

```diff
@@ -52,6 +53,9 @@ protected void setupEnvironment(final TestDestinationEnv environment) throws Exc
         RedshiftSource.DRIVER_CLASS);
 
     schemaName = Strings.addRandomSuffix("integration_test", "_", 5).toLowerCase();
+
+    config = config.set("schemas", Jsons.jsonNode(List.of(schemaName)));
+
     final String createSchemaQuery = String.format("CREATE SCHEMA %s", schemaName);
```

Comment:
Can you also create a second schema (like `unselected_schema`) with some tables, and verify in the acceptance test that the catalog will not include anything in the second schema?

Otherwise, I think the schema selection is not tested any where currently.

Reply:
I just realized that the `SourceAcceptanceTest` does not actually verify the catalog. I created an issue to track it:
https://github.com/airbytehq/airbyte/issues/9818